CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+### 0.0.2 (2022-05-04)
+
+### Bug Fixes
+
+* **vscode-certora-prover:** contract name from user is now added to the "files" section in the conf file ([#60](https://github.com/Certora/vscode-certora-prover/pull/60))
```

Comment:
Let's remove the repository name (we don't plan to change it in the near future).
Also, let's be more specific:
`The main contract name is now...`